app/services/DynamoBannerDesigns.scala

```diff
@@ -232,30 +232,25 @@ class DynamoBannerDesigns(stage: String, client: DynamoDbClient)
     putAllBatched(deleteRequests)
   }
 
-  def updateStatuses(
```

Comment:
updated this function to only take a single designName, which means no need for a batched transaction

Reply:
which has the nice side effect of being much faster!

Reply:
👍  My rational for going straight to bulk setting of statuses was that I could imagine wanting to archive multiple at once down the line (assuming we'd add `archived` to the list of possible statuses) but I was probably overthinking it. Better to keep it simple until we need that functionality.